retreaver-typescript/src/base/Request.ts

```
@@ -0,0 +1,265 @@
+  }
+
+  /**
+   * Post JSON data (alias for getJSON)
+   */
+  postJSON<T = any>(
+    requestUrl: string,
+    payload?: any,
+    callbacks?: Function | Function[],
+    context?: any
+  ): void {
+    return this.getJSON(requestUrl, payload, callbacks, context);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)